test/data/in_tail_chef.yaml

```diff
@@ -0,0 +1,88 @@
+log_file_path: /var/log/ops_agent/ops_agent.log
+logging_level: info
+logs_module:
+  sources:
+  - file_source_config:
+      checkpoint_file: /var/lib/google-fluentd/pos/chef-bookshelf.pos
+      parser:
```

Comment:
If fluentd's format type is 'none', I think the tool doesn't add parser at all. Same for the rest of the file.

Reply:
Oh I didn't know that - will change the logic.